php/mixphp-workerman/composer.json

```
@@ -0,0 +1,11 @@
+{
+  "autoload": {
+    "psr-4": {
+      "App\\": "src/"
+    }
+  },
+  "require": {
+    "workerman/workerman": "~4.0.0",
```

```suggestion
    "workerman/workerman": "*",
```
we don't care about `workerman` version `here`